delivery_multi_destination/tests/test_delivery_multi_destination.py

```diff
@@ -84,9 +84,9 @@ def _create_carrier(self, childs):
         carrier_form = Form(self.env["delivery.carrier"])
         carrier_form.name = "Test carrier multi"
         carrier_form.product_id = self.product
+        carrier_form.destination_type = "multi"
```

Comment:
Why changing the order of this line?

Reply:
Oh, I see that is moving it back to the original position. Now I see the CI red, so maybe it's impacting in something. If it impacts, put a comment saying that this needs to be done in this order. If not impacting, better to neutralize the change in the previous commit, and don't revert it back in your commit.

Reply:
@pedrobaeza done!